openstf/pipeline/train_predict_backtest.py

```diff
@@ -0,0 +1,135 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
```

Comment:
Perhaps `predict` is also fine. But I think we should aim for consistency, up until now we called this processing `create_forecast`. I would suggest either use `train_model_create_forecast_backtest.py` or perhaps just `backtest.py` if we want to use a shorter name.

Reply:
Good point